pyof/v0x04/common/utils.py

```diff
@@ -115,3 +115,14 @@ def new_message_from_header(header):
     message.header.length = header.length
 
     return message
+
+
+def unpack_message(buffer):
+    """Unpack the whole buffer, including header pack."""
+    hdr_size = Header().get_size()
+    hdr_buff, msg_buff = buffer[:hdr_size], buffer[hdr_size:]
+    header = Header()
+    header.unpack(hdr_buff)
+    message = new_message_from_header(header)
```

Comment:
`new_message_from_header` is going to broke, because is not defined here also, right ? 

We have to be careful here, otherwise this is gone to use methods and attributes from v0x01.

@diraol @macartur comments, please.